### PR TITLE
Updated broken development-guide links

### PR DESCRIPTION
**Pull request by jasonodoom**
#  #

## 🗣 Description ##
I updated some broken links in the development guide both in `./dev_envs/mac-env-setup.md` and `onboarding-checklist.md`. 

## 💭 Motivation and context ##

As a new member of the team I stumbled upon these broken links while on-boarding and want to fix them so they're accessible in the future for the next person.
 

## ✅ Pre-approval checklist ##

- [X] This PR has an informative and human-readable title.
- [X] Changes are limited to a single goal - *eschew scope creep!*
- [X] All relevant type-of-change labels have been added.
- [X] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [X] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [X] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Thanks for cleaning up those links! 👍 ⭐

**Review by jsf9k — APPROVED** [review]
Thanks for the updates @jasonodoom!  Please note for the future that commit messages should be in the imperative mood, as is mentioned [here](https://cbea.ms/git-commit/#imperative) (linked to from the development guide [here](https://github.com/cisagov/development-guide/#git-github-and-you)).

File: dev_envs/mac-env-setup.md
Comment (mcdonnnj) [inline dev_envs/mac-env-setup.md]: We should probably update this to mention that these steps are only necessary if you want to manually configure these vs using the Jamf-pushed configuration.

**Review by mcdonnnj — CHANGES_REQUESTED** [review]

File: dev_envs/mac-env-setup.md
Comment (jasonodoom) [reply dev_envs/mac-env-setup.md]: added verbatim.

File: dev_envs/mac-env-setup.md
Comment (mcdonnnj) [inline dev_envs/mac-env-setup.md]:
```suggestion
> **Note**
> These steps are only necessary if you want to manually configure these vs
> using the Jamf-pushed configuration.
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]

File: dev_envs/mac-env-setup.md
Comment (mcdonnnj) [reply dev_envs/mac-env-setup.md]: Added in [31a9e38](https://github.com/cisagov/development-guide/pull/87/commits/31a9e3882f1ae43807effc191baf76d3c148bbd4).

File: dev_envs/mac-env-setup.md
Comment (jsf9k) [inline dev_envs/mac-env-setup.md]: Extremely minor suggestion.  "vs" is an abbreviation for "versus", and in American English abbreviations like this always end in a period.  (British English _does not_ use a period if the last letter of the abbreviation is the last letter of the word, which is also why they write "Mr" instead of "Mr.".) 
```suggestion
These steps are only necessary if you want to manually configure these vs.
```

**Review by jsf9k — COMMENTED** [review]
One extremely minor suggestion.

**Comment by jsf9k** [comment]
@mcdonnnj, we still need your re-review here.

File: dev_envs/mac-env-setup.md
Comment (jasonodoom) [reply dev_envs/mac-env-setup.md]: fair point. fixed!

File: dev_envs/mac-env-setup.md
Comment (mcdonnnj) [inline dev_envs/mac-env-setup.md]: This is a [specific GitHub-flavored Markdown syntax](https://github.com/orgs/community/discussions/16925) that I suggested. Also I would like consensus on if we prefer `vs` or `vs.` as #79 has the former and this has the latter (pre-suggestion).

```suggestion
> **Note**
> These steps are only necessary if you want to manually configure these vs.
> using the Jamf-pushed configuration.
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]

File: dev_envs/mac-env-setup.md
Comment (jsf9k) [reply dev_envs/mac-env-setup.md]: The dot is grammatically correct for American English.

Comment (mcdonnnj) [reply dev_envs/mac-env-setup.md]: Yes I prefer the dot but it was not mentioned in the other PR so I wanted to get a team consensus on preference.

Comment (dav3r) [reply dev_envs/mac-env-setup.md]: Team Dot over here!

Comment (jsf9k) [reply dev_envs/mac-env-setup.md]: I think the three of us makes it an official consensus!

Comment (jsf9k) [reply dev_envs/mac-env-setup.md]: I added the dot to @mcdonnnj's suggested change.

**Comment by jsf9k** [comment]
BTW @jasonodoom, as a member of @cisagov/team-ois you don't have to fork these repos.  You can just clone them, create a new branch, push it up, and create your PR.

File: dev_envs/mac-env-setup.md
Comment (jsf9k) [reply dev_envs/mac-env-setup.md]: @mcdonnnj - please see commits dc38701 and a1831ea.

File: dev_envs/mac-env-setup.md
Comment (jsf9k) [inline dev_envs/mac-env-setup.md]: @jasonodoom - I think the last two bullets are duplicates.  Can you verify and add this change it that's the case?
```suggestion
```

**Review by jsf9k — CHANGES_REQUESTED** [review]

File: dev_envs/mac-env-setup.md
Comment (mcdonnnj) [inline dev_envs/mac-env-setup.md]: Thoughts on using an alternate display for these since it's more of a mandate than a "just for your informatino" item?

```suggestion
> **Warning**
```

File: dev_envs/mac-env-setup.md
Comment (mcdonnnj) [inline dev_envs/mac-env-setup.md]: Same as the previous suggestion.

```suggestion
> **Warning**
```

**Review by jsf9k — APPROVED** [review]

File: dev_envs/mac-env-setup.md
Comment (mcdonnnj) [inline dev_envs/mac-env-setup.md]: Since this repo went private the wiki is no longer available.

```suggestion
[COOL (Cloud-Optimized Operations Lab)](https://github.com/cisagov/cool-system/):
```

File: onboarding-checklist.md
Comment (mcdonnnj) [inline onboarding-checklist.md]: Given the GWE migration should these be updated now since they will soon be broken? There are other Google URLs in here as well I believe.

Comment (dav3r) [reply onboarding-checklist.md]: I think any internal links like this should be removed here and just make this a checklist of items that the new Devvie should ask an existing Devvie about.

Comment (jsf9k) [reply onboarding-checklist.md]: OK, I will make that change.

Comment (jsf9k) [reply onboarding-checklist.md]: Please see commits 11ec021 and 32fcef3 and let me know what you think.

Comment (dav3r) [reply onboarding-checklist.md]: Those both look great to me. 👍

**Review by dav3r — APPROVED** [review]
👍 👍

**Review by dv4harr10 — COMMENTED** [review]
Goto code/development-guide/onboarding-checklist.md line 92 link https://groups.google.com/a/beta.dhs.gov/d/forum/all returns error message 'Content unavailable. Click here to try again'. After click, returns Google 403 error. We're sorry, but you do not have access to this page.

**Comment by dv4harr10** [comment]
The same error results are noticed for links on line 93 thru line 99: - [ ] [CISA COOL Administrators](https://groups.google.com/a/trio.dhs.gov/forum/#!forum/cisa-cool-group)
  - [ ] [cisa-cool-account-support](https://groups.google.com/a/trio.dhs.gov/d/forum/cisa-cool-account-support-group)
  - [ ] [cisa-cool-users](https://groups.google.com/a/trio.dhs.gov/d/forum/cisa-cool-users-group)
  - [ ] [cisagov-github](https://groups.google.com/a/trio.dhs.gov/forum/#!forum/cisagov-github-group)
  - [ ] [NCATS](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats)
  - [ ] [NCATS AWS Support](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats-aws-support)
  - [ ] [NCATS Infrastructure Development](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats-dev)

**Comment by dav3r** [comment]
> Goto code/development-guide/onboarding-checklist.md line 92 link https://groups.google.com/a/beta.dhs.gov/d/forum/all returns error message 'Content unavailable. Click here to try again'. After click, returns Google 403 error. We're sorry, but you do not have access to this page.

> The same error results are noticed for links on line 93 thru line 99..

@dv4harr10 All of those links are internal and you don't have access to them, but more importantly, they were removed via commit https://github.com/cisagov/development-guide/pull/87/commits/32fcef357d26795594a400f688c985d1545f8658, so there is no problem here.

**Comment by jmorrowomni** [comment]
> The same error results are noticed for links on line 93 thru line 99: - [ ] [CISA COOL Administrators](https://groups.google.com/a/trio.dhs.gov/forum/#!forum/cisa-cool-group)
> 
> * [ ]  [cisa-cool-account-support](https://groups.google.com/a/trio.dhs.gov/d/forum/cisa-cool-account-support-group)
> * [ ]  [cisa-cool-users](https://groups.google.com/a/trio.dhs.gov/d/forum/cisa-cool-users-group)
> * [ ]  [cisagov-github](https://groups.google.com/a/trio.dhs.gov/forum/#!forum/cisagov-github-group)
> * [ ]  [NCATS](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats)
> * [ ]  [NCATS AWS Support](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats-aws-support)
> * [ ]  [NCATS Infrastructure Development](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats-dev)

@dv4harr10 That entire section has been removed in commit 32fcef3

**Comment by dv4harr10** [comment]
Thanks for the updates. Also, in the onboarding-checklist.md file line 145 link returns the message ‘This site can’t be reached’.

@ https://jira.ncats.cyber.dhs.gov/secure/Dashboard.jspa

**Comment by dav3r** [comment]
> Thanks for the updates. Also, in the onboarding-checklist.md file line 145 link returns the message ‘This site can’t be reached’.
> 
> @ https://jira.ncats.cyber.dhs.gov/secure/Dashboard.jspa

@dv4harr10 That link was also removed in https://github.com/cisagov/development-guide/pull/87/commits/32fcef357d26795594a400f688c985d1545f8658.  Please ensure that you are looking at the [current version of this pull request](https://github.com/cisagov/development-guide/pull/87/files) before you note any additional issues.

I also encourage you to use GitHub PR review comments to indicate exactly which lines of code you are referring to (see [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) for more info).

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
Approval intensifies!

File: onboarding-checklist.md
Comment (jsf9k) [reply onboarding-checklist.md]: @mcdonnnj, can you let me know if we can resolve this conversation or not?  We are also waiting on a re-review from you for this PR.

File: onboarding-checklist.md
Comment (mcdonnnj) [inline onboarding-checklist.md]: This section should be updated to remove the links since we purged other internal links in this PR.

**Review by mcdonnnj — CHANGES_REQUESTED** [review]

File: onboarding-checklist.md
Comment (jsf9k) [reply onboarding-checklist.md]: Please see commit c066db4.

File: onboarding-checklist.md
Comment (mcdonnnj) [inline onboarding-checklist.md]: Are we considering these internal links to be purged as well? It seemed like were just wholesale axing anything that isn't on GitHub.

File: onboarding-checklist.md
Comment (mcdonnnj) [reply onboarding-checklist.md]: @jsf9k You missed the leading `[` for the link on the line before. Do you want to roll that into the commit you made and force-push?

Comment (jsf9k) [reply onboarding-checklist.md]: Please see commit 58d3788.

File: onboarding-checklist.md
Comment (jsf9k) [reply onboarding-checklist.md]: @cisagov/team-ois, what do you think?

Comment (dav3r) [reply onboarding-checklist.md]: I'm fine getting rid of these since we don't really use them.

Comment (jsf9k) [reply onboarding-checklist.md]: @mcdonnnj, please see commit 30503ef.

File: onboarding-checklist.md
Comment (mcdonnnj) [inline onboarding-checklist.md:1]: The links for the `CISA COOL AWS Accounts` and `Performance Plans` documents are both internal Google links so they should probably be removed. Additionally the wiki is currently off in [cisagov/cool-system](https://github.com/cisagov/cool-system) due to the previous situation around data exposure. Should we either 1) turn on the wiki in that repository or 2) rename the URL's link text to accurately reflect what you'd land on if you visit it?

Comment (dav3r) [reply onboarding-checklist.md:1]: We can't turn on the wiki in cool-system since it's a private repo and we don't pay for that feature.  Also, the COOL Wiki markdown docs are no longer in cool-system anyway, so we might as well just remove that link or else point to the correct private repo.

Comment (jsf9k) [reply onboarding-checklist.md:1]: Please see commits d6b24a5 and 6cf9f5d.

Comment (dav3r) [reply onboarding-checklist.md:1]: Regarding https://github.com/cisagov/development-guide/commit/6cf9f5d1b28d8e85054cb639c222ce40cbf09107,  I think @mcdonnnj was saying that we shouldn't name the link "COOL Wiki" if it's not pointing to an actual wiki anymore- just a wiki-like collection of Markdown files.  I don't have a good suggestion for what we should rename this link to - got any ideas?

Comment (jsf9k) [reply onboarding-checklist.md:1]: Please see commit f96a50d.

Comment (dav3r) [reply onboarding-checklist.md:1]: Works for me. 👍

Comment (mcdonnnj) [reply onboarding-checklist.md:1]: @dav3r Regarding turning on the wiki: I forgot both `cool-system` repos were private (because that makes sense...).

**Review by mcdonnnj — APPROVED** [review]

**Pull request merged**
